tests/test_video.py

```diff
@@ -635,40 +664,65 @@ def test_video_splitter(self, segment_duration: float, mode: str, reset_db: None
                         min_segment_duration=min_segment_duration,
                     ),
                 )
-                self._validate_splitter_segments(t, s, overlap, min_segment_duration)
+                self._validate_splitter_segments(t, s, overlap, min_segment_duration, eps=eps)
                 pxt.drop_table('videos', force=True)
 
+    @pytest.mark.parametrize('segment_times,mode', [([6.0, 11.0, 16.0], 'fast'), ([6.0, 11.0, 16.0], 'accurate')])
+    def test_video_splitter_segment_times(self, segment_times: list[float], mode: str, reset_db: None) -> None:
+        from pixeltable.iterators import VideoSplitter
+
+        eps = 0.1 if mode == 'fast' else 0.0
+        video_filepaths = get_video_files()
+        t = pxt.create_table('videos', {'video': pxt.Video})
+        t.insert([{'video': p} for p in video_filepaths])
+        s = pxt.create_view(
+            'segments', t, iterator=VideoSplitter.create(video=t.video, segment_times=segment_times, mode=mode)
+        )
+        start_times = [0.0, *segment_times]
+        durations = [start_times[i + 1] - start_times[i] for i in range(len(start_times) - 1)]
+        self._validate_splitter_segments(t, s, 0.0, 0.0, expected_durations=durations, eps=eps)
+        pxt.drop_table('videos', force=True)
```

Comment:
why? it'll get cleared out before the next test anyway, and sometimes it can be useful to leave them around for debugging.